doc/cephfs/mount-using-kernel-driver.rst

```diff
@@ -58,50 +58,68 @@ for backporting fixes to their stable kernel: check with your vendor.
 
-  mount -t ceph cephuser@b3acfc0d-575f-41d3-9c91-0e7ed3dbb3fa.cephfs=/ -o mon_addr=192.168.0.1:6789,secret=AQATSKdNGBnwLhAAnNDKnH65FmVKpXZJVasUeQ==
+#. ``name`` is the username of the CephX user we are using to mount CephFS.
+#. ``fsid`` is the FSID of the ceph cluster which can be found using the ``ceph
+   fsid`` command. ``fs_name`` is the file system to mount. The kernel driver
+   requires the MON's socket and the secret key of the CephX user. For example:
 
-When using the mount helper, monitor hosts and FSID are optional. ``mount.ceph``
-helper figures out these details automatically by finding and reading ceph conf
-file, .e.g::
+.. prompt:: bash #
+
+   mount -t ceph cephuser@b3acfc0d-575f-41d3-9c91-0e7ed3dbb3fa.cephfs=/ -o mon_addr=192.168.0.1:6789,secret=AQATSKdNGBnwLhAAnNDKnH65FmVKpXZJVasUeQ==
 
-  mount -t ceph cephuser@.cephfs=/ -o secret=AQATSKdNGBnwLhAAnNDKnH65FmVKpXZJVasUeQ==
+When using the mount helper, monitor hosts and FSID are optional. The
+``mount.ceph`` helper discovers these details by finding and reading the ceph
+conf file. For example:
 
-.. note:: Note that the dot (``.``) still needs to be a part of the device string.
+.. prompt:: bash #
 
-A potential problem with the above command is that the secret key is left in your
-shell's command history. To prevent that you can copy the secret key inside a file
-and pass the file by using the option ``secretfile`` instead of ``secret``::
+   mount -t ceph cephuser@.cephfs=/ -o secret=AQATSKdNGBnwLhAAnNDKnH65FmVKpXZJVasUeQ==
 
-  mount -t ceph cephuser@.cephfs=/ /mnt/mycephfs -o secretfile=/etc/ceph/cephuser.secret
+.. note:: Note that the dot (``.`` in the string ``cephuser@.cephfs``) must  be
+   a part of the device string.
 
-Ensure the permissions on the secret key file are appropriate (preferably, ``600``).
+A weakness of this method is that it will leave the secret key is in your
+shell's command history. To avoid this, copy the secret key inside a
+file and pass the file by using the option ``secretfile`` instead of
+``secret``. For example:
 
-Multiple monitor hosts can be passed by separating each address with a ``/``::
+.. prompt:: bash #
 
-  mount -t ceph cephuser@.cephfs=/ /mnt/mycephfs -o mon_addr=192.168.0.1:6789/192.168.0.2:6789,secretfile=/etc/ceph/cephuser.secret
+   mount -t ceph cephuser@.cephfs=/ /mnt/mycephfs -o secretfile=/etc/ceph/cephuser.secret
 
-In case CephX is disabled, you can omit any credential related options::
+Ensure that the permissions on the secret key file are appropriate (preferably,
+``600``).
+
+Multiple monitor hosts can be passed by separating addresses with a ``/``:
+
+.. prompt:: bash #
+
+   mount -t ceph cephuser@.cephfs=/ /mnt/mycephfs -o
+   mon_addr=192.168.0.1:6789/192.168.0.2:6789,secretfile=/etc/ceph/cephuser.secret
+
+If CephX is disabled, omit any credential-related options. For example:
+
+.. prompt:: bash #
 
-  mount -t ceph cephuser@.cephfs=/ /mnt/mycephfs
+   mount -t ceph cephuser@.cephfs=/ /mnt/mycephfs
```

Comment:
I think this should still specify a mon

Reply:
We're not doing tech stuff on this pass. Each of these commands will be altered, possibly each in its own separate pull request so that we can roll back anything incorrect and so that we have a granular record of each alteration.

Reply:
Harumph.

Reply:
We'll get there. I'll forward you the beefy email that Janne wrote to me about this section. That might reconcile you to this approach.

Reply:
You take me too seriously sir

Reply:
Oh, I don't know about that. I take you seriously, especially as a person who cares about the integrity and accuracy and comprehensibility of the documentation of the Ceph project.